data/hai/hai missions.txt

```diff
@@ -209,6 +209,69 @@ mission "Ask the Hai about the Unfettered"
 
 
 
+mission "Hai Sympathizer Warning"
+	invisible
+	source
+		government "Hai"
+		not attributes "uninhabited" "retirement"
+	to offer
+		"reputation: Hai Merchant (Sympathizers)" < 0
+		not "warned about hai merchant politics"
+	on offer
+		log `Got noticed at a spaceport bar as a "famous warrior." Was warned to look out for bounty hunters or pirates hoping to make a name for themselves.`
+		set "warned about hai merchant politics"
+		conversation
+			`As you sit in a spaceport bar waiting for the food that you've ordered, a Hai unceremoniously pulls up a chair opposite you and takes a seat. He looks to be a merchant, and one who has seen more than a few things besides. He squints at you before speaking, "You are Captain <last>, are you not?"`
+			choice
+				`	"That would be me."`
+					goto acknowledge
+				`	"I think you have the wrong captain."`
+			`	He slaps a device on the table and a small holographic projection of your face springs up and begins rotating around. "No, I think not."`
+			label acknowledge
+			`	He heaves a sigh. "You've been fighting the Unfettered you see, and there are some Hai that don't take too kindly to that. They view you as an outsider sticking your nose into Hai internal politics. Some of those are what we merchants like to call Sympathizers; and they can get a little carried away with themselves and fire off a few shots if they see someone like yourself getting a little too gun-happy where the Unfettered are concerned."`
+			`	With a wave of his hand he indicates the spaceport around you. "You might have noticed some difficulty landing potentially. You see because there are lots of people who don't like outsiders putting their noses in where they're not wanted, if you happen to shoot back and accidentally destroy a Sympathizer's ship, the authorities don't much care who shot first. There are enough of those opinions to ensure that if you shoot down a Sympathizer, it is treated no differently than if you shot down any other Hai merchant. The security forces will treat it the same.`
+			`	"Now the merchants, we're a different matter. It is a topic of great and ongoing debate among us, so you might notice that some people are less offended when you shoot down someone they've been arguing with, but everyone has friends on all sides, so you should watch yourself nonetheless. Sympathizers may become upset enough to shoot at you anywhere they see you. Despite that, the simple fact of the matter is that, if you retaliate against a Hai, any Hai, it'll upset someone and count against you. Even if that Hai was a Sympathizer, and even if they shot first..."`
```

Comment:
I love this whole thing but it is a bit of a text wall, is it not?
Would be good to have a way to break it down a bit

Reply:
It's only 4 lines, and to be honest, he's not interested in letting you ask that question, since it should be obvious by the time he finishes.

Reply:
maybe just say you try to say something but can't quite place a word?
looking at FW for instance even looking at a video had a 'continue watching' "choice"

Reply:
You are being lectured by someone who has come to you because you messed up.
And again, it's at most 4 lines. We didn't break things up in HR unless they were over 5 lines long.

Reply:
Alright then